snuba/snapshots/__init__.py

```diff
@@ -26,6 +27,12 @@ class SnapshotDescriptor:
     id: SnapshotId
     tables: Sequence[TableConfig]
 
+    def get_table(self, table_name: str):
+        for t in self.tables:
+            if t.table == table_name:
+                return t
+        return None
```

Comment:
Why return `None` here instead of raise? Is not finding the table in the dump an expected case or an exceptional one?

Reply:
It is a mistake.